plugin-template

```diff
@@ -30,8 +30,8 @@ DEFAULT_SETTINGS = {
     "coverage": False,
     "deploy_client_to_pypi": True,
     "deploy_client_to_rubygems": True,
-    "deploy_daily_client_to_pypi": True,
-    "deploy_daily_client_to_rubygems": True,
+    "deploy_daily_client_to_pypi": False,
+    "deploy_daily_client_to_rubygems": False,
```

Comment:
I would like to remove the daily client pushing too. I cannot believe someone is using them. I still think we publish bindings with a release.